dashboard/pkg/epinio/pages/c/_cluster/dashboard.vue

```diff
@@ -2,30 +2,47 @@
 import Vue from 'vue';
 import DashboardCard from '../../../components/dashboard/Cards.vue';
 import { createEpinioRoute } from '../../../utils/custom-routing';
-import { EpinioApplicationResource, EpinioCatalogService, EPINIO_TYPES } from '../../../types';
+import {
+  EpinioApplicationResource, EpinioCatalogService, EPINIO_MGMT_STORE, EPINIO_TYPES, METRIC
+} from '../../../types';
 import ConsumptionGauge from '@shell/components/ConsumptionGauge.vue';
 import Namespace from '@shell/models/namespace';
+import { parseSi, createMemoryValues } from '@shell/utils/units';
 import EpinioServiceModel from '../../../models/services';
 import isEqual from 'lodash/isEqual';
 import { sortBy } from 'lodash';
 import { Location } from 'vue-router';
+import Banner from '@components/Banner/Banner.vue';
 
 type ComponentService = {
   name: string,
   link: Location,
   isEnabled: boolean
 }
 
+const threshold = {
+  cpu:    5,
+  memory: 5,
+};
+
 export default Vue.extend<any, any, any, any>({
-  components: { DashboardCard, ConsumptionGauge },
+  components: {
+    Banner,
+    DashboardCard,
+    ConsumptionGauge
+  },
   async fetch() {
     await Promise.all([
       this.$store.dispatch(`epinio/findAll`, { type: EPINIO_TYPES.CATALOG_SERVICE }),
       this.$store.dispatch(`epinio/findAll`, { type: EPINIO_TYPES.NAMESPACE }),
       this.$store.dispatch(`epinio/findAll`, { type: EPINIO_TYPES.SERVICE_INSTANCE })
     ]);
-
     this.version = await this.$store.dispatch('epinio/version');
+    const nodeMetricsSchema = this.$store.getters[`epinio/schemaFor`](METRIC.NODE);
+
+    if (nodeMetricsSchema && !this.$store.getters['isSingleProduct']) {
+      await this.calcAvailableResources();
```

Comment:
is it possible to do this in the same await as the other requests? we should probably sneak version in there too (though i think we should have this already). there's an `allHash` function that will retain the results in an object